CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # Precise UI Changelog
 
 ## 1.0.1
+
 - Fix opening `UploaderProgressDetails` behind modal in IE (#225)
+- Remove refocus of component(Autocomplete) on every value change, remove corresponding test
```

Comment:
I just updated origin. It's no more `1.0.1`. Can you update the version and changelog?

Reply:
Also seems like it is kind of **Breaking Change**, but I wouldn't say it complete change the functionality, it rather fixes the issue, so probably we can bump up the minor version.

Reply:
> Also seems like it is kind of Breaking Change, ...

> ..., so probably we can bump up the minor version

@Saladinek Could you update a minor instead of a patch?